### ISSUE TITLE
S'pht'ia lava at start of level has this weird lighting if you take the secret route and then come back

### ISSUE THREAD

**Issue by aaronfreed** [Pfhorrest/Eternal#327]
it's this lava: 

![image](https://user-images.githubusercontent.com/8217916/83923899-34e78880-a751-11ea-9ab9-d2d9a97f012f.png)

it gets really dark. I think probably no one noticed before because we were using the default bloom values for lava. Once I stepped on the polygon with the recharger, the light changed to being normal.

further miscellaneous observations:

I wonder if there’s a way to fix this view of the tower using partially untextured walls:

![image](https://user-images.githubusercontent.com/8217916/83923992-737d4300-a751-11ea-8b70-dd3bdaab031c.png)

but that would mean the game would look really stupid for anyone not using the shader renderer

because of a bug in the engine, the upper halves of these platform walls do this weird texture glitch that results from two “extends from both” platforms sharing a boundary:

![image](https://user-images.githubusercontent.com/8217916/83924177-f2727b80-a751-11ea-9dc5-7222bfd01f9e.png)

might be worth redrawing the boundaries to add *more* platforms, with tiny little platforms in between them that don’t change their heights when they activate. i think this would fix the texture glitch

this is too difficult to get out of; add a canister or invincibility powerup or something

![image](https://user-images.githubusercontent.com/8217916/83924698-387c0f00-a753-11ea-957f-4cb1ea506dd1.png)

(i cheesed it using invincibility through the console)

lights are weirdly aligned:

![image](https://user-images.githubusercontent.com/8217916/83927306-81839180-a75a-11ea-8fa4-646401293c5e.png)

(this is true for all of these)
might be worth modifying the size slightly with the assistance of the grid function in Weland, so that it conforms to some multiple of .125

**Comment by aaronfreed** [comment]
fix the solid lines here:

![image](https://user-images.githubusercontent.com/8217916/83966193-c60c4b80-a886-11ea-9dd3-51616c69d664.png)

i know this platform was super slow in the original game, but still, speed it up:

![image](https://user-images.githubusercontent.com/8217916/83966222-efc57280-a886-11ea-894f-4d889ce83c4b.png)

(look for other slow platforms on this map too)

outdoor texture alignment:

![image](https://user-images.githubusercontent.com/8217916/83966297-62365280-a887-11ea-9d53-cc3e2a4f2a9f.png)

![image](https://user-images.githubusercontent.com/8217916/83966311-7bd79a00-a887-11ea-9e78-815a5c4961df.png)

![image](https://user-images.githubusercontent.com/8217916/83966321-8eea6a00-a887-11ea-9269-20c07adf457c.png)

![image](https://user-images.githubusercontent.com/8217916/83966334-a32e6700-a887-11ea-9955-77c887fb4cad.png)

basically check all of these

bouncy walls @ window, but kind of minor, maybe not even worth fixing:

![image](https://user-images.githubusercontent.com/8217916/83966400-07512b00-a888-11ea-9556-b1f77b0dbdcd.png)

might try to add window here too:

![image](https://user-images.githubusercontent.com/8217916/83966472-64e57780-a888-11ea-8463-4acdc7a01756.png)
